htpasswd.go

```diff
@@ -122,7 +122,7 @@ func (h *HtpasswdFile) Reload() error {
 	users := make(map[string]string)
 	for _, record := range records {
 		if !validUsernameRegexp.MatchString(record[0]) {
-			log.Printf("Ignoring invalid username %q in htpasswd, consists of characters other than letters", record[0])
+			log.Printf("Ignoring invalid username %q in htpasswd, consists of characters other than letters, number, '_', '-', '.' and '@'", record[0])
```

Comment:
Hate me for being picky but if you could amend the commit after s/number/numbers/ ..